jarek_ciolek/challange_2.py

```diff
@@ -0,0 +1,17 @@
+import random
+input_list = [random.randint(-1,1) for i in range(100)]
+
+def api_call(input_list):
+    '''This function takes a list of values and then it returns a list with the
+    same number of elements [True, False,...] where
+    True if element >= 0
+    False if element < 0'''
+    
+    new_list = []
+    for i in input_list:
+    	if input_list[i] >= 0:
```

Comment:
`if i >= 0:`
you are looping over a list, `i` is not the index, but each element.